dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTableTest.java

```diff
@@ -0,0 +1,121 @@
+package com.dotmarketing.startup.runonce;
+
+import static org.junit.Assert.assertTrue;
+
+import com.dotcms.datagen.ContainerDataGen;
+import com.dotcms.datagen.FolderDataGen;
+import com.dotcms.datagen.LinkDataGen;
+import com.dotcms.datagen.TemplateDataGen;
+import com.dotcms.datagen.TestDataUtils;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.beans.Identifier;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.IdentifierAPI;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.containers.model.Container;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.folders.model.Folder;
+import com.dotmarketing.portlets.links.model.Link;
+import com.dotmarketing.portlets.templates.model.Template;
+import com.dotmarketing.util.UtilMethods;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+
+public class Task201014UpdateColumnsValuesInIdentifierTableTest {
+
+    private static IdentifierAPI identifierAPI;
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6327767)